packages/lib/actionClass/auth.ts

```diff
@@ -0,0 +1,21 @@
+import { hasUserEnvironmentAccess } from "../environment/auth";
+import { getActionClass } from "./service";
+import { unstable_cache } from "next/cache";
+
+export const canUserAccessActionClass = async (userId: string, actionClassId: string): Promise<boolean> =>
```

Comment:
I think it makes sense to validate inputs here, too (like in services)

Reply:
done